src/components/ui/Input/FileInput.tsx

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React, { Ref, forwardRef } from 'react';
+
+import { InputProps } from './';
+import { StyledInput } from './Styled';
+
+interface FileInputProps extends InputProps {
+  optionText?: String;
+}
+
+export const FileInput = forwardRef(
+  (props: FileInputProps, ref: Ref<HTMLInputElement>) => {
+    const { id, optionText, onChange } = props;
```

Comment:
We should be spreading props so developers can pass their own props when needed (also means you don't need to pull out `onChange` explicitly)

Reply:
Done.